docs/release-notes/v1.1.rst

```diff
@@ -1,6 +1,178 @@
 .. _gammapy_1p1_release:
 
-1.0 (unreleased)
-----------------
+1.1 (May XXth 2023)
+-------------------
 
-- No changes yet
+- Released May XXth, 2023
+- 14 contributors
+- 69 pull requests since v1.0.1 (not all listed below)
```

Comment:
should we not count from 1.0.0 ?

Reply:
I have used the last release as reference but all PRs with 1.1 milestones have been taken into account (but not he bug fixes that went to 1.0.1).

Reply:
I don't know what other packages do, but I would naively imagine that the last *feature* release could be taken as reference, and bug fixes counted in *both* bug fix and feature releases